legal-api/src/legal_api/resources/business/business_filings.py

```diff
@@ -73,7 +74,7 @@ def get(identifier, filing_id=None):  # pylint: disable=too-many-return-statemen
             if str(request.accept_mimetypes) == 'application/pdf' and filing_id:
                 if rv.filing_type == 'incorporationApplication':
                     return legal_api.reports.get_pdf(rv.storage, None)
-            filing_json = rv.json
+            filing_json = rv.raw if original_filing else rv.json
             filing_json['filing']['documents'] = DocumentMetaService().get_documents(filing_json)
```

Comment:
Do we want to include the list of documents on the raw json, as we may not be able to render it directly anymore?

Reply:
So when user request for original filing we don't need to include documents. isn't it?

Reply:
Yes, I don't think that should be included. It should just be the raw json originally submitted by the customer.

We won't be "back" supporting outputs, as those will always be created using the versioned data in simple filing.

Reply:
ok